### ISSUE TITLE
How could we improve the Open Food Facts API and its documentation so that it's easier to implement in the Flutter package?

### ISSUE THREAD

**Issue by stephanegigandet** [openfoodfacts/openfoodfacts-dart#473]
I'm posting this as an issue as it's the easiest way to reach the contributors of the openfoodfacts-dart package, we can close this issue in a couple of weeks.

Open Food Facts has a Google Summer of Docs intern, Funke Olasupo, who is working on improving the Open Food Facts API documentation. As you have been implementing it in the Flutter package, it would be great to have your feedback and ideas!

Could you help by filling this form https://forms.gle/qzTKMLxRUEakwQcj7 ?

"Hello team, we will be conducting a survey to know more about your issues and difficulties with the current documentation. I know I already reached out to some persons for a brief call , however if you feel there's more you need to share please use this form.

If you also think you want to explain your difficulties on a call, I am open :yeux_en_cœur: please send a DM on slack:+1:"

**Comment by monsieurtanuki** [comment]
(open) Food for thought: https://documentation.divio.com/?mc_cid=5dcae0a22e&mc_eid=186ee87e7b

TL;DR
> There is a secret that needs to be understood in order to write good software documentation: there isn’t one thing called documentation, there are four.
They are: tutorials, how-to guides, technical reference and explanation. They represent four different purposes or functions, and require four different approaches to their creation. Understanding the implications of this will help improve most documentation - often immensely.

**Comment by teolemon** [comment]
@stephanegigandet ok to close ?
